src/sentry/options/defaults.py

```diff
@@ -128,6 +128,10 @@
 register("slack-v2.client-secret", flags=FLAG_PRIORITIZE_DISK)
 register("slack-v2.signing-secret", flags=FLAG_PRIORITIZE_DISK)
 
+# Slack migration
+register("slack-migration.enable-banner-check", default=False)
+register("slack-migration.banner-sample-rate", default=0.0)
```

Comment:
We don't need a sample rate, the risk is extremely low and we don't use this in any production capacity.

Reply:
@mattrobenolt When you saw the risk is extremely low, do you mean the risk that we could be adding excessive DB load? And what do you mean we don't use this in any production capacity?

Reply:
Just for clarificaion since it's going to be mostly irrelevant:

The endpoint is hit very low volume, and the impact is very low of it failing. By having a knob here to crank up the rate is going to yield no results. The endpoint is hit asynchronously on page load. I don't even think it continuously polls. So the database wouldn't even see the affects of this, and numbers will be way too low. Even if the endpoint failed, it'd do nothing to the UI except not show your message.

So having sample rate isn't worth doing. It'll go from 0 -> 1 only. We wouldn't do it in steps.

Reply:
Also my comment on "production capacity" was becasue this endpoint is only shown to superusers thus my original thinking of this only being for system administrators so the volume would be even lower + not even applicable to SaaS.

Sorry, all of my comments here were based on the fact that this endpoint is only accessible to superusers, so that framed all of my context accordingly.